src/Sylius/Component/Attribute/spec/AttributeType/SelectAttributeTypeSpec.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Attribute\AttributeType;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Attribute\AttributeType\AttributeTypeInterface;
+
+/**
+ * @author Laurent Paganin-Gioanni <l.paganin@algo-factory.com>
+ */
+class SelectAttributeTypeSpec extends ObjectBehavior
```

Comment:
I forget to add the "final" in my last commit, but not to overload travis i will push it this night.